my-app/src/redux/game/reducers.js

```diff
@@ -0,0 +1,43 @@
+import calculateWinner from '../../app/utils//utils';
```

Comment:
I wonder how `//` works 🤔

Reply:
good catch! that should be a /utils, might be a typo.